test/metabase/api/setup_test.clj

```diff
@@ -127,8 +128,11 @@
                                        :first_name             first-name
                                        :last_name              last-name
                                        :email                  email
-                                       :user_group_memberships [{:id 1} {:id 2}]}}
-                           logged-event)))))))))))))
+                                       :user_group_memberships [{:id (:id (perms-group/all-users))}]}}
+                           logged-event))))
+                  (testing "created user is admin"
```

Comment:
this is subtle; previously, we published `event/user-invited` twice, so the event being tested here is the 2nd audit -- which we removed because it's redundant.

And since the permissions are set after the user is created, the audit event does not include admin as part of the user's group.

Reply:
Subtle indeed! Would be nice to "fix" the audit log, but seems like that would need more of a hack than it's worth.

Reply:
so I decided to fix this so it does not change the existing behavior: 943cf77425